src/shared/schemas/v10/index.js

```diff
@@ -0,0 +1,60 @@
+import map from 'lodash/map';
+import each from 'lodash/each';
+import merge from 'lodash/merge';
+import v9Schema from '../v9';
+
+const migration = (oldRealm, newRealm) => {
+    const walletData = newRealm.objectForPrimaryKey('Wallet', 9);
+    const newWalletSettings = newRealm.objects('WalletSettings');
+    const oldNotificationsSettings = oldRealm.objects('NotificationsSettings');
+    const newNotificationsSettings = newRealm.objects('NotificationsSettings');
+    // Bump wallet version.
+    walletData.version = 10;
+
+    each(newWalletSettings, (settings) => {
+        settings.hideEmptyTransactions = true;
+    });
+
+    each(oldNotificationsSettings, (oldSettings) => {
+        each(newNotificationsSettings, (newSettings) => {
+            newSettings.general = oldSettings.general;
+            newSettings.confirmations = oldSettings.confirmations;
+            newSettings.messages = false;
+        });
+    });
```

Comment:
```suggestion
    each(oldNotificationsSettings, (oldSettings) => {
        oldSettings.messages = newNotificationSettings.messages;
    });
```
I might be missing something, but are we not able to do this instead? Putting an `each` inside another `each` would cause the inner `each` to run 3 times

Reply:
You are correct. Good idea.